src/app/components/AppContainer/startupProcessSteps/saveStorageInforamationFactory.ts

```diff
@@ -0,0 +1,16 @@
+import useStorage from '@/app/store/useStorage';
+import type { Dispatch } from '@/app/store';
+import type { StartupMessage } from '@/types/AsyncMessages';
+
+export function saveStorageInforamationFactory(
```

Comment:
```suggestion
export function saveStorageInformationFactory(
```

should probably change any occurrence of this typo